project-handbook/technical-infrastructure.md

```diff
@@ -0,0 +1,63 @@
+
+For this GitHub organisation, we distribute permissions using Teams and a repository that follows a four-eyes and version-controlled paradigm. Please see [this document](./github-management.md) for guidance and details.
+
+In addition to the official GitHub organisation, there is a playground organisation with lower governance standards: [OpenRail-Playground](https://github.com/OpenRail-Playground). Only non-official projects must be hosted there. Permissions are controlled individually using the GitHub web interface.
+
+## Mails and Mailing Lists
+
+OpenRail Association runs a number of email addresses via which people can reach us and individual teams. We don't run a mail server ourselves but use two intertwined services: the [MailEasy product by our domain registrar INWX](https://www.inwx.de/en/hosting) and mailing lists by [Mailman3.com](https://mailman3.com), a provider of mailing lists using the Open Source software [Mailman 3](https://mailman3.org).
+
+The MailEasy product as part of INWX's hosting services doesn't provide mail boxes but only forwards. This way, we can configure certain addresses such as `technical-committee@openrailassociation.org` to forward received emails to another address. In some instances, this is one or multiple persons individually (e.g. the Technical Infrastructure Manager), but more often its a larger group of people.
+
+In these cases, we address these people using mailing lists. This allows for more fine-grained configuration of who can contact this group, and has advantages when dealing with anti-spam techniques such as SPF, DKIM and DMARC which often trigger when just using forwards.
+
+There are a number of mailing lists we've set up on our mailing lists instance [lists.openrailassociation.org](https://lists.openrailassociation.org). As some of them shall be reachable by outside people, we've set up some forwards from the openrailassociation.org domain to certain lists.openrailassociation.org lists. One example is `technical-committee@openrailassociation.org` which forwards to `technical-committee@lists.openrailassociation.org`.
+
+When setting up new lists, make sure to have a look at the list configuration of lists which have the same or a similar use-case. The default settings are often not sufficient to make all email reach all people, especially regarding the top-level domain aliases and anti-spam measurements.
+
+## Passwords
+
+For the several services we are using, some require dedicated accounts and passwords, while others are ties to existing user accounts, e.g. GitHub.
+
+If we have to share passwords, we are using an hosted password vault. The Technical Infrastructure Manager manages this vault and provides access to other people, like their assistants, on a need-to-know basis. A sub-part of this vault may also be used by other teams if they require such a services. However, we generally prefer using pre-existing, personalised accounts.
```

Comment:
It's a vaultwarden service but not written in stone. We're still evaluating, it may also become the official Bitwarden paid instance if it's easier to use for all participants (e.g. because they're using it in other contexts as well and it integrates into their workflows better than another service).